UNITY-7762 add isproductnumbervisible (#1231)
ePages-de/epages-docs: apps/data-types.md

| productDataSheet | string | An image or PDF file containing a data sheet with technical information on the product. Has to be available if the product has an energy label. |
| sfUrl | string | The link to the storefront URL of the product. |
| productNumber | string | The product number. |

> | isProductNumberVisible | boolean | Indicates if the product number can be shown in the storefront. |

Reviewer: ```suggestion
| isProductNumberVisible | boolean | Indicates if the product number is displayed in the storefront. |
```

| productDataSheet | string | An image or PDF file containing a data sheet with technical information on the product. Has to be available if the product has an energy label. |
| sfUrl | string | The link to storefront URL of the product. |
| productNumber | string | The product number. |

> | isProductNumberVisible | boolean | Indicates if the product number can be shown in the storefront. |

Reviewer: ```suggestion
| isProductNumberVisible | boolean | Indicates if the product number is displayed in the storefront. |
```
